### PR TITLE
Welcome log

### PR DESCRIPTION
**Pull request by Lyst67**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -29,6 +32,7 @@ const ErrorPage = lazy(() => import('./pages/ErrorPage/ErrorPage'));
 
 function App() {
   const isLoggedIn = useSelector(selectIsLoggedIn);
+  const isParamsData = useSelector(selectIsParamsData);
```
Comment (Sokilskill) [inline src/App.jsx]: const isParams

File: src/redux/auth/auth-operations.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import { createAsyncThunk } from '@reduxjs/toolkit';
 import axios from 'axios';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
-axios.defaults.baseURL = 'https://power-pulse-6-backend.onrender.com/api/';
+axios.defaults.baseURL = 'https://power-pulse-6-backend.onren.com/api/';
```
Comment (Sokilskill) [inline src/redux/auth/auth-operations.js]: url змінилась?

**Review by Sokilskill — APPROVED** [review]

**Comment by Lyst67** [comment]
По url вибач. Випробовував помилку і забув повернути.

19 січня 2024, 11:27:03, від "Serhii" ***@***.***>:


@Sokilskill approved this pull request.
In src/App.jsx:
> @@ -29,6 +32,7 @@ const ErrorPage = lazy(() => import('./pages/ErrorPage/ErrorPage'));
 
 function App() {
   const isLoggedIn = useSelector(selectIsLoggedIn);
+  const isParamsData = useSelector(selectIsParamsData);
const isParams
In src/redux/auth/auth-operations.js:
>  
-axios.defaults.baseURL = 'https://power-pulse-6-backend.onrender.com/api/';
+axios.defaults.baseURL = 'https://power-pulse-6-backend.onren.com/api/';
url змінилась?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

--=-ijZsN7JQc998YL5im/t4
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: base64

PGh0bWw+PGJvZHk+PHNwYW4gc3R5bGU9ImRpc3BsYXk6YmxvY2s7IiBjbGFzcz0ieGZtXzIzNzgx
MTgxIj48ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6MTJwdDtsaW5lLWhlaWdodDoxNHB0O2Zv
bnQtZmFtaWx5OkFyaWFsOyIgY2xhc3M9InhmbWMxIj7Qn9C+IHVybCDQstC40LHQsNGHLiDQktC4
0L/RgNC+0LHQvtCy0YPQstCw0LIg0L/QvtC80LjQu9C60YMg0ZYg0LfQsNCx0YPQsiDQv9C+0LLQ
tdGA0L3Rg9GC0LguPC9zcGFuPjxici8+PC9kaXY+DQo8ZGl2Pjxici8+PC9kaXY+DQo8ZGl2Pjxp
PjxzcGFuIHN0eWxlPSJmb250LXNpemU6MTBwdDtsaW5lLWhlaWdodDoxMnB0OyI+PHNwYW4gc3R5
bGU9ImZvbnQtZmFtaWx5OkFyaWFsOyI+MTkg0YHRltGH0L3RjyAyMDI0LCAxMToyNzowMywg0LLR
ltC0DQogICAgIlNlcmhpaSIgJmx0Ozwvc3Bhbj48YSBocmVmPSJtYWlsdG86bm90aWZpY2F0aW9u
c0BnaXRodWIuY29tIiB0YXJnZXQ9Il9ibGFuayI+PHNwYW4gc3R5bGU9ImZvbnQtZmFtaWx5OkFy
aWFsOyI+bm90aWZpY2F0aW9uc0BnaXRodWIuY29tPC9zcGFuPjwvYT48c3BhbiBzdHlsZT0iZm9u
dC1mYW1pbHk6QXJpYWw7Ij4mZ3Q7Og0KPC9zcGFuPjwvc3Bhbj48L2k+PC9kaXY+DQo8ZGl2Pjxi
ci8+PC9kaXY+DQo8YmxvY2txdW90ZSBzdHlsZT0iYm9yZGVyLWxlZnQ6MXB4IHNvbGlkICNjY2Nj
Y2M7bWFyZ2luOjBweCAwcHggMHB4IDAuOGV4O3BhZGRpbmctbGVmdDoxZXg7Ij4NCjxkaXYgc3R5
bGU9ImRpc3BsYXk6YmxvY2s7Ij4NCjxkaXY+PC9kaXY+DQo8ZGl2PjxiPkBTb2tpbHNraWxsPC9i
PiBhcHByb3ZlZCB0aGlzIHB1bGwgcmVxdWVzdC48L2Rpdj4NCjxoci8+PGRpdj5JbiA8YSBocmVm
PSJodHRwczovL2dpdGh1Yi5jb20vU29raWxza2lsbC9wcm9qZWN0LWRhaWx5LWVuZXJneS1kb3Nl
L3B1bGwvMzMjZGlzY3Vzc2lvbl9yMTQ1ODYzNTY3NiIgdGFyZ2V0PSJfYmxhbmsiIHJlbD0ibm9y
ZWZlcnJlciBub29wZW5lciI+c3JjL0FwcC5qc3g8L2E+OjwvZGl2Pg0KPHByZSBzdHlsZT0ibWFy
Z2luOjVweCAwOyI+Jmd0OyBAQCAtMjksNiArMzIsNyBAQCBjb25zdCBFcnJvclBhZ2UgPSBsYXp5
KCgpID0mZ3Q7IGltcG9ydCgnLi9wYWdlcy9FcnJvclBhZ2UvRXJyb3JQYWdlJykpOw0KIA0KIGZ1
bmN0aW9uIEFwcCgpIHsNCiAgIGNvbnN0IGlzTG9nZ2VkSW4gPSB1c2VTZWxlY3RvcihzZWxlY3RJ
c0xvZ2dlZEluKTsNCisgIGNvbnN0IGlzUGFyYW1zRGF0YSA9IHVzZVNlbGVjdG9yKHNlbGVjdElz
UGFyYW1zRGF0YSk7DQo8L3ByZT4NCjxkaXY+Y29uc3QgaXNQYXJhbXM8L2Rpdj4NCjxoci8+PGRp
dj5JbiA8YSBocmVmPSJodHRwczovL2dpdGh1Yi5jb20vU29raWxza2lsbC9wcm9qZWN0LWRhaWx5
LWVuZXJneS1kb3NlL3B1bGwvMzMjZGlzY3Vzc2lvbl9yMTQ1ODY0NTUwMSIgdGFyZ2V0PSJfYmxh
bmsiIHJlbD0ibm9yZWZlcnJlciBub29wZW5lciI+c3JjL3JlZHV4L2F1dGgvYXV0aC1vcGVyYXRp
b25zLmpzPC9hPjo8L2Rpdj4NCjxwcmUgc3R5bGU9Im1hcmdpbjo1cHggMDsiPiZndDsgIA0KLWF4
aW9zLmRlZmF1bHRzLmJhc2VVUkwgPSAnaHR0cHM6Ly9wb3dlci1wdWxzZS02LWJhY2tlbmQub25y
ZW5kZXIuY29tL2FwaS8nOw0KK2F4aW9zLmRlZmF1bHRzLmJhc2VVUkwgPSAnaHR0cHM6Ly9wb3dl
ci1wdWxzZS02LWJhY2tlbmQub25yZW4uY29tL2FwaS8nOw0KPC9wcmU+DQo8ZGl2PnVybCDQt9C8
0ZbQvdC40LvQsNGB0Yw/PC9kaXY+DQo8ZGl2PjxzcGFuIHN0eWxlPSJmb250LXNpemU6c21hbGw7
Y29sb3I6IzY2NjsiPuKAlDxici8+UmVwbHkgdG8gdGhpcyBlbWFpbCBkaXJlY3RseSwgPGEgaHJl
Zj0iaHR0cHM6Ly9naXRodWIuY29tL1Nva2lsc2tpbGwvcHJvamVjdC1kYWlseS1lbmVyZ3ktZG9z
ZS9wdWxsLzMzI3B1bGxyZXF1ZXN0cmV2aWV3LTE4MzE5NTcyODMiIHRhcmdldD0iX2JsYW5rIiBy
ZWw9Im5vcmVmZXJyZXIgbm9vcGVuZXIiPnZpZXcgaXQgb24gR2l0SHViPC9hPiwgb3IgPGEgaHJl
Zj0iaHR0cHM6Ly9naXRodWIuY29tL25vdGlmaWNhdGlvbnMvdW5zdWJzY3JpYmUtYXV0aC9BTzM1
VUs1S0dEVFpPM0o3REtCTVJSM1lQSTM2TkFWQ05GU002QUFBQUFCQ0FRWFdZV1ZISTJEU01WUVdJ
WDNMTVY0M1lVRFZOUldGRVpMUk9WU1hHNUNTTVYzR1NaTFhITVlUUU1aUkhFMlRPTVJZR00iIHRh
cmdldD0iX2JsYW5rIiByZWw9Im5vcmVmZXJyZXIgbm9vcGVuZXIiPnVuc3Vic2NyaWJlPC9hPi48
YnIvPllvdSBhcmUgcmVjZWl2aW5nIHRoaXMgYmVjYXVzZSB5b3UgYXV0aG9yZWQgdGhlIHRocmVh
ZC48aW1nIHNyYz0iaHR0cHM6Ly9naXRodWIuY29tL25vdGlmaWNhdGlvbnMvYmVhY29uL0FPMzVV
SzJQWE9CRkxGTVJFTzRBTEFEWVBJMzZOQTVDTkZTTTZBQUFBQUJDQVFYV1lXV0dHMzNOTlZTVzQ1
QzdPUjRYQVpOUktCMldZM0NTTVZZWEtaTFRPUkpHSzVUSk1WMzJVWTNQTlZXV0szVFVMNVVXSlRU
TkdGWlNHLmdpZiIgaGVpZ2h0PSIxIiB3aWR0aD0iMSIgYWx0PSIiLz48L3NwYW4+PHNwYW4gc3R5
bGU9ImZvbnQtc2l6ZTowO2NvbG9yOnRyYW5zcGFyZW50O2Rpc3BsYXk6bm9uZTtvdmVyZmxvdzpo
aWRkZW47d2lkdGg6MDtoZWlnaHQ6MDttYXgtd2lkdGg6MDttYXgtaGVpZ2h0OjA7Ij5NZXNzYWdl
IElEOiAmbHQ7U29raWxza2lsbC9wcm9qZWN0LWRhaWx5LWVuZXJneS1kb3NlL3B1bGwvMzMvcmV2
aWV3LzE4MzE5NTcyODNAZ2l0aHViLmNvbSZndDs8L3NwYW4+PC9kaXY+DQo8L2Rpdj4NCjwvYmxv
Y2txdW90ZT48L3NwYW4+PC9ib2R5PjwvaHRtbD4NCg==

--=-ijZsN7JQc998YL5im/t4--

**Comment by Sokilskill** [comment]
ок

**Comment by Sokilskill** [comment]
не стягуй собі з гіта, а запуш і ще раз пулл реквест своїх змінених файлів

**Pull request merged**